apps/wallet/src/modals/FirstTimePopup/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { modalActions, Text } from '@walless/gui';
+import { BlueCircleBackground } from '@walless/icons';
+import { ModalId } from 'modals/types';
+import { mockWidgets } from 'state/widget';
+import { navigate } from 'utils/navigation';
+import { addWidgetToStorage } from 'utils/storage';
+
+import PixeverseCard from './PixeverseCard';
+
+const FirstTimePopup = () => {
+	const handleAddPixeverse = () => {
+		addWidgetToStorage(mockWidgets[0]._id, mockWidgets[0]);
+		navigate('Dashboard', {
+			screen: 'Explore',
+			params: {
+				screen: 'Widget',
+				params: {
+					id: mockWidgets[0]._id,
```

Comment:
I don't think you should use the `[0]` index to query for `Pixeverse` widget, should use a concrete pixeverse id instead